app/public/js/common/configLocation.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs-extra');
+const userHome = require('user-home');
+
+class Configuration {
+
+    /**
+     * Get the configuration path
+     * 
+     * @returns {string} The location of the config
+     */
+    static get path () {
+        let userConfigPath = null;
+
+        /** Linux platforms - XDG Standard */
+        if (process.platform === 'win32') {
```

Comment:
I do not believe so, This was written according to the nodejs documentation for [process.platform](https://nodejs.org/api/process.html#process_process_platform) which only lists win32 (I think it refers to the win32 api, not windows 32bit)